src/lib/journey/callbacks/terms-and-conditions/terms-conditions.svelte

```diff
@@ -41,4 +47,5 @@
 
 <Checkbox {firstInvalidInput} key={inputName} onChange={setValue} value={false}>
   <T key="termsAndConditions" />
+  <Message classes="tw_col-start-2 tw_row-start-2" dirtyMessage={termsLinkHtml} />
```

Comment:
Just a reference to a good resource about this a11y concern that led to not placing the link in label: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#accessibility_concerns.